client/src/components/Plugins/Store/PluginAuthForm.tsx

```diff
@@ -23,10 +23,10 @@ function PluginAuthForm({ plugin, onSubmit }: TPluginAuthFormProps) {
           className="col-span-1 flex w-full flex-col items-start justify-start gap-2"
           method="POST"
           onSubmit={handleSubmit((auth) =>
-            onSubmit({ pluginKey: plugin!.pluginKey, action: 'install', auth }),
+            onSubmit({ pluginKey: plugin?.pluginKey, action: 'install', auth }),
```

Comment:
has this been tested with a plugin with/without authentication needed?

Reply:
I have tried installing and uninstalling the plugin with and without authentication, and it works